apis/courseprogressapi/courseprogressapi.yaml

```diff
@@ -166,25 +166,29 @@ definitions:
     type: object
     properties:
       userId:
-        description: 'This ID uniquely identifies a user'
+        description: 'This Id uniquely identifies a user'
         example: Sam123
         type: string
       courseIds:
-        description: 'This represents the list of course IDs'
+        description: 'This represents the list of course IDs, pass only one id incase you are passing contentIds as well. Multiple courseIds will be consider if user need to fetch data for different courseId irrespective of any batch and content.'
         example:
           - some course ID
         type: array
         items:
           type: string
       contentIds:
-        description: 'This represents the list of content IDs for that course'
+        description: 'This represents the list of content IDs for that course and batch.Pass one courseId and batchId and list of all content ids.'
```

Comment:
One or more content IDs corresponding to specified course ID. Specify only one course ID when specifying content IDs.